src/content/blog/filecoin-foundation-quarterly-update-october-2024.md

```diff
@@ -19,13 +19,13 @@ seo:
 
 Twenty years ago, Web2 ushered in greater interactivity and collaboration on the web. It also gave rise to a few dominant players across the web who centralized control over networks and information online. Today, artificial intelligence (AI) is at risk of following a similar, centralized path. Major AI developers are [locking down their data and code](https://www.vice.com/en/article/5d3naz/openai-is-now-everything-it-promised-not-to-be-corporate-closed-source-and-for-profit), leaving builders and consumers with little visibility into how models function. The integrity of any AI model relies heavily on transparency around how it’s developed, what data it relies on, and how it operates. Yet, the current trend toward centralization and opacity threatens to [erode global trust in AI](https://assets.kpmg.com/content/dam/kpmg/au/pdf/2023/trust-in-ai-global-insights-2023.pdf).
 
-As the AI industry continues to expand, driving an increasing need for data storage and computational power, Web3 tools like Filecoin are uniquely positioned to help decentralize and support AI development. According to a recent [Grayscale Research](https://www.grayscale.com/research/reports/ai-is-coming-crypto-can-help-make-it-right) report on the intersection of crypto and AI, “decentralized AI holds the potential to bring these important decisions out from walled gardens and into public ownership.”
+As the AI industry continues to expand, driving an increasing need for data storage and computational power, Web3 tools like Filecoin are uniquely positioned to help decentralize and support AI development. According to a recent [Grayscale Research]() report on the intersection of crypto and AI, “decentralized AI holds the potential to bring these important decisions out from walled gardens and into public ownership.”
```

Comment:
No more link here? Or is it for testing purposes?

Reply:
Never mind, it is for testing purposes https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/793/commits/4fff16935298e39a44fdca125cb5aed08d83ed93